app/src/index.css

```diff
@@ -8,36 +8,105 @@
 
 /* Add styles to the base styles: https://tailwindcss.com/docs/adding-base-styles */
 @layer base {
-  #app-main {
-    @apply text-gray-700; /* Default font color */
+  #app {
+    @apply text-grey-500; /* Default font color */
+    @apply text-sm md:text-md lg:text-lg xl:text-xl 2xl:text-xxl;
+    font-kerning: auto;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    text-rendering: optimizeLegibility;
+    scroll-behavior: smooth;
+  }
+
+  h1 {
+    @apply text-headline;
+  }
+
+  a {
+    @apply underline;
+    @apply hover:text-grey-400;
+  }
+
+  [type="text"],
+  [type="email"],
+  [type="url"],
+  [type="password"],
+  [type="number"],
+  [type="date"],
+  [type="datetime-local"],
+  [type="month"],
+  [type="search"],
+  [type="tel"],
+  [type="time"],
+  [type="week"],
+  [multiple],
+  textarea,
+  select {
+    @apply w-full p-4 border border-grey-400;
+    @apply hover:border-grey-500;
+    @apply focus:outline-none focus:border-grey-500 focus:ring-0;
+    font-size: 100%;
   }
 }
 
 /* Add styles to components */
 @layer components {
+  .indent {
+    text-indent: 1em;
+  }
```

Comment:
> What is this class for? Indenting paragraphs? Maybe some inline comments about when to use various classes would help clear this up

indenting paragraphs is a typographical classic thing for text - e.g. a project description - see in figma on the grant detail page.

other iconpacks : definitly no need. if we miss a custom icon, i will just add it to fusion.


breaking what ever flexbox classes from not-my-code css/html of the page seems to be ok for me - as it will be replaced anyway with my code .  the code fragments for the cart are ready to use .

getting fusion to a higher version number - could graham do as he pushed it to npm for me. 

writing from vacation :)

Reply:
Thanks @scco! Now get back to enjoying vacation 😛